typings-checker/global-id.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { globalId } from "../dist/src/index";
+
+const { getRootId, getLocalId } = globalId;
+
+describe('Global ID', () => {
+    describe('getRootId', () => {
```

Comment:
missing some positive examples - `getRootId({id:'foo'})`, `getRootId(new MyComponent())` etc.

yeah, the unit tests makes that redundant in most cases, but when there's a typings test I think we should include the positive alongside the negative.